Sources/Bot/Models/RequiredStatusCheck.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+public struct RequiredStatusChecks: Decodable, Equatable {
+    /// Require branches to be up to date before merging?
+    public let strict: Bool
+    /// Names of status checks marked as Required
+    public let contexts: [String]
```

Comment:
Definition (Doc-comment) of `strict` and `contexts` look quite overlapping to me,
so we might want to simplify `struct RequiredStatusChecks` , possibly by using enum instead.

Reply:
They are two independent concepts, branches can have a list of checks marked as required that must pass to enable merging but they can also have an option to force PRs to be up to date with the base branch and they are not mutually exclusive, you can have just one of them enabled, both or even none.

Reply:
OK I was a bit confused due to these params order.

`contexts` should probably come first as a required param, and `strict` comes secondary as a possibly optional param (which can add default value e.g. `true`), so changing order will improve readability in this case.

If we read `strict` first, `contexts` also seem strict required params that overlaps previous keyword.